source/usage-examples/updateMany.txt

```diff
@@ -3,3 +3,60 @@ Update Multiple Documents
 =========================
 
 .. default-domain:: mongodb
+
+You can update multiple documents in a collection by using the ``UpdateMany()``
+method.
+
+The following example passes a query filter and an update parameter to the
+``UpdateMany()`` method, which matches documents in the
+``listingsAndReviews`` collection if the value of their ``address.market``
+field is "Sydney" and multiplies the ``price`` field in the matching
+documents by a factor of *1.15*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateMany.go
+   :start-after: begin updatemany
+   :end-before: end updatemany
+   :emphasize-lines: 5
+   :language: go
+   :dedent:
+
+Click here **<TODO: github link to file>** to see a fully runnable example.
+
+Expected Result
+---------------
+
+After you run the code example, the ``price`` field of the matched
+documents will reflect the multiplication. The updated documents will
+appear similar to these:
+
+.. code-block:: json
+   :copyable: false
+
+   // results truncated
+   { 
+      "_id" : "10091713", ... , "name" : "Surry Hills Studio", ... , "price": 181.00, ...
+   }
+   { 
+      "_id" : "9908871", ... , "name" : "Family friendly beach house", ... , "price": 751.00, ...
+   }
+   { 
+      "_id" : "20989061", ... , "name" : "Big and sunny Narraben room", ... , "price": 60.00, ...
+   }
```

Comment:
```suggestion
   ...
   { "_id" : "10091713", ... , "name" : "Surry Hills Studio", ... , "price": 181.00, ... }, 
   { "_id" : "9908871", ... , "name" : "Family friendly beach house", ... , "price": 751.00, ...  }, 
   {  "_id" : "20989061", ... , "name" : "Big and sunny Narraben room", ... , "price": 60.00, ... }, 
   ...
```
Depending on if there are documents before or after, there should be `...` to show theres more that you're not including.

Reply:
accepted!